package.json

```diff
@@ -37,10 +38,12 @@
     "@boxyhq/saml-jackson": "file:./npm",
     "cors": "2.8.5",
     "next": "12.0.7",
+    "pako": "^2.0.4",
```

Comment:
what is pako needed for?

Reply:
I didn't notice, probably i was doing some experiment.